docs/how-to/backup-and-restore.md

```diff
@@ -0,0 +1,129 @@
+# How to back up and restore Synapse
+
+This document shows how to back up and restore Synapse.
+
+A backup strategy is an integral part of disaster and recovery, and should be
+planned accordingly. The main goal of a backup is the possibility of being
+restored and the backup and restore process should be tested as part of
+normal operations.
+
+The process of backing up and restoring depends on whether an external database
+is used, so the step to run the backup for PostgreSQL must be done only if PostgreSQL
+is used in the original Synapse application.
+
+## Back up Synapse
+
+### Deploy s3-integrator charm
+
+Synapse gets backed up to a S3 compatible object storage. The bucket for the backup should be provisioned before the backup.
+
+For Synapse to get the credentials, the `s3-integrator` is used. Refer to [s3-integrator](https://charmhub.io/s3-integrator/) for specific configuration options. 
+
+```
+juju deploy s3-integrator --channel edge
+juju config s3-integrator endpoint=<s3 endpoint> bucket=<bucket name> path=<optional-path> region=<region> s3-uri-style=<path or host>
+juju run s3-integrator/leader sync-s3-credentials access-key=<access-key> secret-key=<secret-key>
+```
+
+Integrate with Synapse with:
+
+`juju integrate synapse:backup s3-integrator`
+
+### Configure the passphrase
+
+The backup will be encrypted before being sent using symmetric encryption. You need
+to set the desired password with:
+```
+juju config synapse backup_passphrase=<secret passphase>
+```
+
+### Create the backup
+
+Create the backup with the next command:
+```
+juju run synapse/leader create-backup
+```
+
+A new object should be placed in the S3 compatible object storage. This file is a tar
+file encrypted with the `gpg` command.
+
+
+You can list your available with the `list-backups` command:
+```
+juju run synapse/leader list-backups
+```
+
+### Back up PostgreSQL
+
+Follow the instructions of the PostgreSQL charm:
+ - For [postgresql-k8s](https://charmhub.io/postgresql-k8s/docs/h-create-and-list-backups).
+ - For [postgresql](https://charmhub.io/postgresql/docs/h-create-and-list-backups).
+
+If you plan to restore PostgreSQL in a different model of cluster, you will need
+to also back up the cluster passwords. See:
+ - For [postgresql-k8s](https://charmhub.io/postgresql-k8s/docs/h-migrate-cluster-via-restore).
+ - For [postgresql](https://charmhub.io/postgresql/docs/h-migrate-cluster-via-restore).
+
+
+## Restore
+
+The recommendation is to first restore PostgreSQL if necessary. Then deploying,
+configuring and integrating Synapse with other charms as in a normal deployment
+and finally restoring Synapse. 
+
+The revisions used PostgreSQL and Synapse charms should be the same ones as the ones used
```

Comment:
s/revisions used PostgreSQL and Synapse charms/the PostgreSQL and Synapse charm revisions